go.mod

```diff
@@ -4,8 +4,11 @@ go 1.21
 
 require (
 	github.com/creasty/defaults v1.7.0
+	github.com/emersion/go-sasl v0.0.0-20200509203442-7bfe0ed36a21
+	github.com/emersion/go-smtp v0.20.1
 	github.com/goccy/go-yaml v1.11.2
 	github.com/icinga/icingadb v1.1.1-0.20230418113126-7c4b947aad3a
+	github.com/jhillyerd/enmime v1.0.1
```

Comment:
There was a new release: https://github.com/jhillyerd/enmime/releases/tag/v1.1.0

Please update now, otherwise we'd just get a dependabot PR anyways.